src/pages/About/AboutKyberSwap.tsx

```diff
@@ -150,7 +148,7 @@ export const KSStatistic = () => {
         <ForTraderInfoRow>
           <ForTraderInfoCell>
             <Text fontWeight="600" fontSize="24px">
-              {MAINNET_NETWORKS.length}
+              12+
```

Comment:
```suggestion
              {MAINNET_NETWORKS.length-1}+
```

Reply:
When adding new chain, everyone gonna miss places like this. Keep it reference to network const to keep it automatically updated.

Reply:
Still hard for others to know why minus 1

Reply:
write down your reason to a comment next to it

Reply:
I don't have a reason...

Reply:
then no need. I see that `{...- 1}+` is very meanful

Reply:
Ok. Done now